src/lib/vip-import-validate-files.js

```diff
@@ -411,3 +418,31 @@ export const logErrorsForIntermediateImages = obj => {
 	}
 	console.log( '------------------------------------------------------------' );
 };
+
+export const summaryLogs = ( folderErrorsLength, intImagesErrorsLength, fileTypeErrorsLength, filenameErrorsLength, totalFiles, totalFolders ) => {
```

Comment:
Minor feedback

In general, for functions with more than 2 arguments, it's hard to remember the order of arguments. Another way of doing this is by accepting an object as an argument with named attributes:

```js
export const summaryLogs = ( { folderErrorsLength, intImagesErrorsLength, fileTypeErrorsLength, filenameErrorsLength, totalFiles, totalFolders } )
```

When calling the function, it's easier than remembering the order:
```js
summaryLogs( {
  folderErrorsLength: folderValidation.length,
  intImagesErrorsLength: intermediateImages.tally,
  fileTypeErrorsLength: errorFileTypes.length,
  filenameErrorsLength: errorFileNames.length,
  totalFiles: files.length,
  totalFolders: nestedDirectories.length
} )
```

Reply:
Gah I meant to do this but forgot! Thank you! Fixed in 1abd712!